tests/system/feature_store/test_spark_engine.py

```diff
@@ -797,30 +798,29 @@ def test_get_offline_features_with_filter(self):
             run_config=fs.RunConfig(local=False),
         )
         assert measurements.status.targets[0].run_id is not None
-
         fv_name = "measurements-fv"
         features = [
             "measurements.bad",
             "measurements.department",
         ]
-
         my_fv = fs.FeatureVector(
             fv_name,
             features,
-            description="my feature vector",
         )
+        my_fv.spec.with_indexes = True
         my_fv.save()
         target = ParquetTarget("mytarget", path=self.get_remote_pq_target_path())
-        fs.get_offline_features(
+        resp = fs.get_offline_features(
             fv_name,
             target=target,
             query="bad>6 and bad<8",
-            engine="spark",
```

Comment:
I took over this test because it doesn't actually do what it purports to do. `get_offline_features` does not currently support `engine="spark"` (see [ML-2802](https://jira.iguazeng.com/browse/ML-2802)).